### DISCUSSION TITLE
Issue with shadows around borders

### DISCUSSION THREAD

**Discussion by TheCreatorAMA** [nvimdev/lspsaga.nvim#925] [General]
Hello, I have a question about UI styling.

Whenever a pop up window appears (this is from hover doc) I get this shadowing below:
![image](https://user-images.githubusercontent.com/64107039/222202389-f5310d46-29b4-4cb4-aab6-ef8fd4bb6f08.png)

But what I don't get is that for outgoing_calls pop up window only part of the window has the shadowing (green bubble shows no shadow and red bubble shows shadow)
![Screenshot 2023-03-01 at 9 38 58 AM](https://user-images.githubusercontent.com/64107039/222204518-9f8a784a-bef2-4934-8804-e5b1e49c5145.png)

This may be due to my lack of understanding on how to configure the ui correctly. Some helpful info below possibly:
- neovim version: 0.8.3
- tokyonight colorscheme (I can send its config if needed)

**Comment by glepnir** [comment]
fixed

**Reply by TheCreatorAMA** [reply]
A follow up question, if I wanted to get rid of that shadow around the border how can I do that?

**Reply by glepnir** [reply]
why don't use solid if you don't want see shadow ?  you used shadow and you want get rid of shadow?

**Reply by TheCreatorAMA** [reply]
Sorry, I could have explained better before. I am using the rounded border option but a shadow is still appearing. I am trying to get rid of it and only have the border line show.

![image](https://user-images.githubusercontent.com/64107039/222455913-a9b0e3b2-42ce-4ecc-904b-352c89a36a7d.png)

Basically, Im trying to see why there is that block around the rounded border line.

**Reply by glepnir** [reply]
it's border background color i think not shadow. set it to same as your theme background
